docs/using/event-types.md

```diff
@@ -44,6 +44,69 @@ The custom schema for an event type can be as simple as
 `{ "\additionalProperties\": true }` to allow arbitrary JSON, but will usually
 have a more specific definition :)
 
+### Compatibility modes
+
+Event type schemas evolve through time and different data integration
+patterns require different guarantees when it comes to what changes
+should or should not be allowed.
+
+To accomodate these different needs Nakadi provides three distinct
```

Comment:
You can also remove `distinct` (only if you want to), the sentence still works without it